eve/main.yml

```diff
@@ -385,7 +385,10 @@ models:
       haltOnFailure: true
   - ShellCommand: &terraform_apply
       name: Spawn openstack virtual infra
-      command: terraform apply -auto-approve
+      command: |-
+        for _ in $(seq 1 3); do
+            terraform apply -auto-approve && break
+        done
```

Comment:
Let's add a few minutes of sleep time between attempts

Reply:
Yes, why not.